docs/library/jesd204/axi_jesd204_rx/index.rst

```diff
@@ -0,0 +1,1129 @@
+     - |  Decoder selection of the link layer:
+       |  1 - 8B/10B mode;
+       |  2 - 64B/66B mode.
+   * - DATA_PATH_WIDTH
+     - Data path width in bytes. Set it to 4 in case of 8B/10B, 8 in case of
+       64B/66B
+
+JESD204 RX Synthesis Configuration Parameters
+--------------------------------------------------------------------------------
+
+.. hdl-parameters::
+   :path: library/jesd204/jesd204_rx
+
+   * - TPL_DATA_PATH_WIDTH
+     - Data path width in bytes towards transport layer. Must be greater or
+       equal to ``DATA_PATH_WIDTH``. Must be a power of 2 integer multiple of
+       the F parameter.
+   * - ASYNC_CLK
+     - Set this parameter to 1 if the link clock and the device clocks have
+       different frequencies, or if they have the same frequency but a
+       different source. If set, synchronizing logic and a gearbox of ratio
+       ``DATA_PATH_WIDTH``:``TPL_DATA_PATH_WIDTH`` is inserted to do the rate
+       conversion. If not set, ``TPL_DATA_PATH_WIDTH`` must match
+       ``DATA_PATH_WIDTH``, the same clock must be connected to ``clk`` and
+       ``device_clk`` inputs.
+
+AXI JESD204 RX Signal and Interface Pins
+--------------------------------------------------------------------------------
+
+.. hdl-interfaces::
+
+    * - s_axi_aclk
+      - All ``S_AXI`` signals and ``irq`` are synchronous to this clock.
+    * - s_axi_aresetn
+      - Resets the internal state of the peripheral.
+    * - s_axi
+      - Memory mapped AXI-lite bus that provides access to modules register map.
+    * - irq
+      - Interrupt output of the module. Is asserted when at least one of the
+        modules interrupt is pending and enabled.
+    * - device_clk
+      - :dokuwiki:`Device clock <resources/fpga/peripherals/jesd204/jesd204_glossary#clocks>` 
+        for the JESD204 interface. Its frequency must be link clock
+        \* ``DATA_PATH_WIDTH`` / ``TPL_DATA_PATH_WIDTH``
+    * - device_reset
+      - Reset active high synchronous with the 
+        :dokuwiki:`Device clock <resources/fpga/peripherals/jesd204/jesd204_glossary#clocks>`.
+
+JESD204 RX Signal and Interface Pins
+--------------------------------------------------------------------------------
+
+.. hdl-interfaces::
+    :path: library/jesd204/jesd204_rx
+
+    * - clk
+      - :dokuwiki:`Link clock <resources/fpga/peripherals/jesd204/jesd204_glossary#clocks>` 
+        for the JESD204 interface. Must be line clock/40 for correct operation 
+        in 8B/10B mode, line clock/66 in 64B/66B mode.
+    * - reset
+      - Reset active high synchronous with the 
+        :dokuwiki:`Link clock <resources/fpga/peripherals/jesd204/jesd204_glossary#clocks>`
+    * - rx_data
+      - Received data.
+    * - sync
+      - sync[m-1:0] is JESD204 SYNC~ (or SYNC_N) signals, available in 8B/10B mode.
+        (``0 <= m < NUM_LINKS``)
+    * - sysref
+      - JESD204 SYSREF signal.
+    * - rx_phy*
+      - n-th lane of the JESD204 interface (``0 <= n < NUM_LANES``).
```

Comment:
Probably more of a doctools comment, but:

This does generate what I think is intended, but there is still a warning during the docs build:
```WARNING: Signal rx_phy* defined in the directive does not exist in the IP-XACT (component.xml)!```

Additionally, it would be nice to have some other way of showing these kinds of interfaces without listing every possible lane.

Reply:
Never mind, I had the wrong version of doctools. This is already fixed :)